src/app/components/normal-component/server-page/server-page.component.spec.ts

```diff
@@ -5,76 +5,64 @@ import { AuthUser } from 'src/app/services/AuthenticationService';
 import { AuthenticationServiceMock } from 'src/app/services/tests/AuthenticationService.spec';
 import { SimpleComponentTestUtils } from 'src/app/utils/tests/TestUtils.spec';
 import { Router } from '@angular/router';
-import { GameService } from 'src/app/services/GameService';
-import { ICurrentPartId } from 'src/app/domain/icurrentpart';
 import { PartMocks } from 'src/app/domain/PartMocks.spec';
+import { ActivePartsService } from 'src/app/services/ActivePartsService';
+import { BehaviorSubject } from 'rxjs';
+import { IPartId } from 'src/app/domain/icurrentpart';
 
 describe('ServerPageComponent', () => {
 
     let testUtils: SimpleComponentTestUtils<ServerPageComponent>;
-    let gameService: GameService;
     let component: ServerPageComponent;
 
     beforeEach(fakeAsync(async() => {
         testUtils = await SimpleComponentTestUtils.create(ServerPageComponent);
         AuthenticationServiceMock.setUser(AuthUser.NOT_CONNECTED);
         component = testUtils.getComponent();
-        gameService = TestBed.inject(GameService);
     }));
     it('should create', fakeAsync(async() => {
         expect(component).toBeDefined();
         component.ngOnInit();
     }));
-    it('should rely on game service to create online games', fakeAsync(async() => {
-        const gameService: GameService = TestBed.inject(GameService);
-        spyOn(gameService, 'createGameAndRedirectOrShowError');
-        AuthenticationServiceMock.setUser(AuthenticationServiceMock.CONNECTED);
-        component.ngOnInit();
-
-        component.pickGame('Awale');
-        component.createGame();
+    it('should dispatch to online-game-selection component when switching to create game tab', fakeAsync(async() => {
```

Comment:
should display online-game-selection component when clicking on tab-create element ?